Chore: Node 18.17.1, Chrome 118 internal images (#1009)
cypress-io/cypress-docker-images: browsers-internal/node18.17.0-chrome118-ff115/README.md

> # cypress/browsers-internal:node18.15.0-chrome116-ff115

Reviewer: ```suggestion
# cypress/browsers-internal:node18.15.0-chrome118-ff115
```

> A complete image with all operating system dependencies for Cypress, and Chrome 118.0.5993.117-1, Firefox 115.0.1

Reviewer: ```suggestion
A complete image with all operating system dependencies for Cypress, and Chrome 118.0.5993.117-1, Firefox 115.0.1 browsers.
```

> A complete image with all operating system dependencies for Cypress, and Chrome 116.0.5845.187-1, Firefox 115.0.1, Edge undefined browsers.

Reviewer: ```suggestion
A complete image with all operating system dependencies for Cypress, and Chrome 118.0.5993.117-1, Firefox 115.0.1, Edge undefined browsers.
```
